test/UnitWOPIAsyncUpload_Close.cpp

```diff
@@ -53,7 +53,7 @@ class UnitWOPIAsyncUpload_Close : public WopiTestServer
             LOK_ASSERT_EQUAL(std::string("false"), request.get("X-COOL-WOPI-IsAutosave"));
 
             // Fail with error.
-            return Util::make_unique<http::Response>(http::StatusLine(404));
+            return Util::make_unique<http::Response>(http::StatusCode::InternalServerError);
```

Comment:
This is intentionally changed to 500 because 404 implies the token is invalid, which means subsequent uploads won't happen. This test isn't trying to invalidate tokens, so best to have a generic server error instead.